sync/handlers/block_request_test.go

```diff
@@ -167,7 +167,7 @@ func TestBlockRequestHandlerLargeBlocks(t *testing.T) {
 	blocks, _, err := core.GenerateChain(gspec.Config, genesis, engine, memdb, 96, 0, func(i int, b *core.BlockGen) {
 		var data []byte
 		switch {
-		case i < 32:
+		case i <= 32:
```

Comment:
What does this do?

Reply:
Causes [this test](https://github.com/ava-labs/coreth/pull/423/files#diff-ff24a27eb19d31cab32d1ffea0260444286838e6a3b56c279594b72b60f2e9baL200-L204) to serialize a large block (1MB).
Without this change, the test would serialize a small block, then skip adding the large block to the output, which is why we didn't detect this issue before.